src/clr-angular/forms/password/password-container.ts

```diff
@@ -118,7 +118,11 @@ export class ClrPasswordContainer implements DynamicWrapper, OnDestroy {
   }
 
   controlClass() {
-    return this.controlClassService.controlClass(this.invalid, this.addGrid());
+    return this.controlClassService.controlClass(
+      this.invalid,
+      this.addGrid(),
+      this.layoutService ? this.layoutService.labelSize : null
```

Comment:
We should be able to just pass `this.layoutService.labelSize` to all of these calls to `controlClass()` to simplify.

Reply:
the reason I chose this logic is because the tests are failing when, for instance, a password container is used outside a clrForm

Reply:
Ahh yeah good call.